prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/DefaultFilterTests.kt

```diff
@@ -320,7 +320,7 @@ class DefaultFilterTests {
             engine.qualityFilterDefault,
             bundle,
             false
-        )
+        ).first
```

Comment:
See my other comment, but it seems like instead of just testing that the first part of the pair is what we expect, we need to update all these tests to test the whole pair?